docs/src/main/sphinx/release/release-398.md

```diff
@@ -0,0 +1,64 @@
+# Release 398 (28 Sep 2022)
+
+## General
+
+* Add Hudi connector. ({issue}`10228`)
```

Comment:
Need to decide if we want to add this here or announce when docs PR is merged @martint

Reply:
Or add "experimental" or some other qualifier